plugins/woocommerce-admin/client/two-column-tasks/task-list-item.tsx

```diff
@@ -123,8 +123,8 @@ export const TaskListItem: React.FC< TaskListItemProps > = ( {
 					action={ () => {} }
 					actionLabel={ task.actionLabel }
 					{ ...props }
-					onClick={ () => {
-						if ( task.isDisabled ) {
+					onClick={ ( e: React.ChangeEvent ) => {
+						if ( task.isDisabled || e.target.tagName === 'A' ) {
```

Comment:
Again, preventing click action when ancestor link is clicked.

Reply:
Nice fix 👍 
The addition of `e: React.ChangeEvent` does give a TypeScript error for me. I was able to fix it by rebasing (Moon's latest fix) and changing this line https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/experimental/src/experimental-list/task-item/index.tsx#L42
to:
`onClick: React.MouseEventHandler< HTMLElement >;`